src/pages/api/account/refresh-tag.js

```diff
@@ -0,0 +1,36 @@
+import HTTP_CODE from "@/lib/constants/httpStatusCodes";
+import { makeBanner } from "@/lib/riitag/banner";
+import logger from "@/lib/logger";
+import { ncWithSession } from "@/lib/routing";
+
+async function refreshTag(request, response) {
```

Comment:
Maybe we should rate-limit this? You can see how it's done in the update-tag stuff. You can use the username as a key before even hitting the database.